test/basic.js

```diff
@@ -235,11 +303,30 @@ test('validate needs to load', (t) => {
             db.publish(post2, (err, msg2) => {
               t.error(err, 'no err')
               t.equal(msg.key, msg2.value.previous)
-              sbot.close(t.end)
+              t.end()
             })
           })
         })
       })
     })
   })
 })
+
+test('publishAs classic', (t) => {
+  const keys = ssbKeys.generate()
+
+  const content = { type: 'post', text: 'hello world!' }
+
+  db.publishAs(keys, content, (err, msg) => {
```

Comment:
A better test case here is to add 2 messages and check that sequence on the second is correct. Just found a bug in the other branch related to that problem ;)